run_wav.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Run as script:
+
+$ python run_wav.py --codec=LSB_wav \
+    --input=data/fox.wav \
+    --output=tmp/fox_secret.wav \
+    --payload=s3cr3t
```

Comment:
```suggestion
$ python run_wav.py --codec=lsb_wav \
    --input=data/moonlight.wav \
    --output=tmp/moonlight_secret.wav \
    --payload=s3cr3t
```

第一行那个 `lsb_wav`，下面 `__init__.py` 里改了这里没改。